Kentico/custom-element-devkit
# DCN-103 - Improve dev kit (value / enabled persistence, proper elemen…

…t URL, element media in output folder)

### Motivation

Fixing several flaws:
* It wasn't possible to test all various combinations of initial selector state (disabled / enabled vs. with value / without value) - Both value and enabled / disabled is persisted
* Relative URLs didn't work properly because of the extension-less nature of the element wrapper
* Added allow-forms sandbox attribute (is now present also on production)

### How to test

Load some element in wrapper (some without initial value JSON)
* Try to refresh the page with enabled / disabled - State should be persisted
* Set some value, refresh the page with enabled / disabled - State should be persisted
* Clear the persisted value - Element should get to a previous state
